packages/cli-dxns/src/dxos.proto

```diff
@@ -0,0 +1,79 @@
+syntax = "proto3";
+
+package dxos;
+
+message DXNS_cid {
```

Comment:
`DXNS_CID`

Reply:
Isn't CID an IPFS standard -- DXNS should just be a content addressable record store using CIDs.
And DXNs map onto CID. So I don't see a need to specialize these.